crates/router/src/core/payment_methods/cards.rs

```diff
@@ -4986,31 +5089,19 @@ where
     not(feature = "payment_methods_v2")
 ))]
 pub async fn get_card_details_with_locker_fallback(
-    pm: &payment_method::PaymentMethod,
+    pm: &domain::PaymentMethod,
     state: &routes::SessionState,
-    key_store: &domain::MerchantKeyStore,
+    _key_store: &domain::MerchantKeyStore,
```

Comment:
We can remove function parameters that are not accessed in both v1 and v2 implementations. You can do the same change for other functions in this file that don't access `key_store`.